app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(names: str) -> None:
+    command, old_file, new_file = names.split(" ")
```

Comment:
According to the checklist, you should ensure that the input string contains exactly three elements separated by spaces, and the first element should be 'cp'. Consider adding a validation step to check this condition.